internal/seclang/directives.go

```diff
@@ -774,9 +774,6 @@ func directiveSecAuditLogRelevantStatus(options *DirectiveOptions) error {
 // Syntax: SecAuditLogParts [PARTLETTERS]
 // Default: ABCFHZ
 // ---
-// The format of the audit log format is documented in detail in the Audit Log Data
-// Format Documentation.
```

Comment:
Is `Audit Log Data Format Documentation` somewhere? If so, we should point it here. I can't find it and the same line looks to be copy pasted from ModSec doc